board/xilinx/zynq/board.c

```diff
@@ -296,9 +296,9 @@ int board_late_init(void)
   }
 
   /* Set serial number environment variable */
-  char buf[16];
-  snprintf(buf, sizeof(buf), "%d", factory_params.serial_number);
-  setenv("serial_number", buf);
```

Comment:
Need to update https://github.com/swift-nav/u-boot-xlnx/blob/swift-v2016.1/include/configs/piksiv3_dev.h#L190